### ISSUE TITLE
Allow pagination with Trending endpoint

### ISSUE THREAD

**Issue by VictorienDruon** [blakejoy/tmdb-ts#45]
Hi, thank you for your efforts on this project.

I've noticed that it's currently not possible to pass a 'page' parameter with the Trending endpoint. This limitation hinders the use of pagination. Although it's not explicitly documented in the TMDB documentation, I've found that we can successfully pass a 'page' parameter in the URL with the trending endpoints, and it functions correctly. The current data returned also includes the necessary information to enable this functionality (e.g., total_pages, page, etc.).

Therefore would it be possible to add a 'page' parameter to the Trending endpoint?

Thanks in advance, and keep up the excellent work!

**Comment by blakejoy** [comment]
Yup that's definitely possible @VictorienDruon . I can work on getting it done or you can make a Pull Request if you would like?
